### PR TITLE
feat: Add welcome view when no folder opened

### PR DESCRIPTION
**Pull request by jdneo**
Address https://github.com/microsoft/vscode-java-pack/issues/540

### CODE REVIEW

**Review by Eskibear — APPROVED** [review]
👍

File: package.nls.json [hunk 1]
```diff
@@ -31,5 +31,8 @@
...
     "configuration.java.test.config.env.description": "Specify the extra environment variables when running the tests",
-    "configuration.java.test.config.sourcePaths.description": "Specify extra source paths when debugging the tests"
+    "configuration.java.test.config.sourcePaths.description": "Specify extra source paths when debugging the tests",
+    "contributes.viewsWelcome.inLightWeightMode": "No test cases are listed because the Java Language Server is currently running in [LightWeight Mode](https://aka.ms/vscode-java-lightweight). To show test cases, click on the button to switch to Standard Mode.\n[Switch to Standard Mode](command:java.server.mode.switch?%5B%22Standard%22,true%5D)",
+    "contributes.viewsWelcome.noProjectWithProjectManagerInstalled": "No folder opened in Visual Studio Code. You can [open a Java project](command:_java.project.open), or create a new Java project by clicking the button below.\n[Create Java Project](command:java.project.create)",
+    "contributes.viewsWelcome.noProjectWithOutProjectManagerInstalled": "No folder opened in Visual Studio Code."
```
Comment (Eskibear) [inline package.nls.json:37]: Can we provide a shortcut to open a folder? Maybe it's not that feasible as commands are different per os.

Comment (jdneo) [reply package.nls.json:37]: We cannot since things in package.json are static.

Comment (Eskibear) [reply package.nls.json:37]: so I'm really interested in how vscode implements that.

Comment (jdneo) [reply package.nls.json:37]: https://github.com/microsoft/vscode/blob/413963c489fafa5163b5d6513731c7953de07fb3/src/vs/workbench/contrib/files/browser/explorerViewlet.ts#L297-L312

Comment (Eskibear) [reply package.nls.json:37]: I just created a issue to request a unified command id:
https://github.com/microsoft/vscode/issues/117748

**Pull request merged**